controllers/cert.go

```diff
@@ -106,6 +106,27 @@ func (c *ApiController) GetGlobleCerts() {
 	}
 }
 
+// GetCertsByScope
+// @Title GetCertsByScope
+// @Tag Cert API
+// @Description get certs by scope
+// @Param   owner     query    string  true        "The owner of the cert"
+// @Param   scope     query    string  true        "The scope of the cert"
+// @Success 200 {object} []object.Cert The Response object
+// @router /get-scope-certs [get]
+func (c *ApiController) GetCertsByScope() {
```

Comment:
we can use getCerts with field = 'scope' and value instead creating new method?

Reply:
fixed